docs/module-auth.AuthenticationClient.html

```diff
@@ -2305,7 +2305,7 @@ <h5>Example</h5>
   scope: 'openid'  // Optional field.
 };
 
-auth0.oauth.token(data, function (err, userData) {
+auth0.oauth.passwordGrant(data, function (err, userData) {
```

Comment:
This file is auto generated based on our source code. The line you want to change is: https://github.com/auth0/node-auth0/blob/master/src/auth/index.js#L512. Any manual changes to the docs files will be lost on the next time we generate the docs.

However, as we aren't calling `oauth` directly, but proxying it through `auth0`, I think we should also use `auth0.passwordGrant` to be consistent.

Once you made those changes, you can run `npm run jsdoc:generate` to generate the docs.

Reply:
Can you run the docs again ? Sorry about that.